constellations/gemini-admin/src/help/help.sass

```diff
@@ -1,24 +1,24 @@
 @import "../_.constants.sass"
 
-body
-  margin: 0
+#view-help
   h1
+    color: $colorTurquoise
     margin-left: 100px
     margin-top: 30px
     font-family: $fontFamilyRoboto
     font-size: $fontSize4
-    color: $colorTurquoise
-    
   p
     margin: 20px 0px 40px 100px
     width: 1000px
     font-family: $fontFamilyRoboto
     color: $colorGrayDark
-    
   h2
     font-family: $fontFamilyRoboto
     font-size: $fontSize4
     margin-left: 100px
     padding-bottom: 20px
     border-bottom: 1px solid
     width: 1000px
+
+    
+
```

Comment:
hmm i wonder if this is being done by your text editor? Can you remove the extra lines?